apps/backoffice-v2/src/common/components/atoms/RouteError/RouteErrorWithProviders.tsx

```diff
@@ -0,0 +1,10 @@
+import { Providers } from '@/common/components/templates/Providers/Providers';
+import { RouteError } from '@/common/components/atoms/RouteError/RouteError';
+
+export const RouteErrorWithProviders = () => {
```

Comment:
Previously places that used the `useCustomerQuery` had no access to react query's provider.